### PR TITLE
Shortened code by 5 lines for memory efficiency

### PR DESCRIPTION
**Pull request by JackyLei94**
*Changed 5 lines for async runtime improvement

### CODE REVIEW

**Comment by JackyLei94** [comment]
ayoooo wassup lmao

File: src/Code.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
+// HeHhaehhwhawhw this is aTest
+// tybigbo1k3vv3rwvbigojinjii8equalsequalsD
+/* oOoO gettin fancy w the comments 
+no neesanno ur irstblezettairyouiki
+5 lines TEST ADDED
```
Comment (kevinjihwanlee) [inline src/Code.js:5]: nerd

**Review by kevinjihwanlee — CHANGES_REQUESTED** [review]
Test PR by friend

File: src/Code.js (hunk 1 above)
Comment (JackyLei94) [reply src/Code.js:5]: it's been good knowin ya

**Pull request closed without merging**